### PR TITLE
Correction of evaluation metric

### PR DESCRIPTION
**Pull request by MELSunny**
tested on pytorch==1.2.0
ISIC 2018 dataset first loop result[All default]:
Epoch [1/150], Loss: 570.2478, 
[Training] Acc: 0.8824, SE: 0.7623, SP: 0.9695, PC: 0.7704, F1: 0.6987, JS: 0.5823, DC: 0.6987
[Validation] Acc: 0.8595, SE: 0.5696, SP: 0.9477, PC: 0.7414, F1: 0.5573, JS: 0.4587, DC: 0.5573
Best R2AttU_Net model score : 1.0160

tested on pytorch==1.5.0
ISIC 2018 dataset first loop result[All default]:
Epoch [1/200], Loss: 454.9332, 
[Training] Acc: 0.9083, SE: 0.7878, SP: 0.9756, PC: 0.8060, F1: 0.7414, JS: 0.6310, DC: 0.7414
[Validation] Acc: 0.8288, SE: 0.3145, SP: 0.9982, PC: 0.6754, F1: 0.3814, JS: 0.2991, DC: 0.3814

### CODE REVIEW

**Comment by MELSunny** [comment]
Correction of evaluation metric
tested on pytorch==1.2.0 and pytorch==1.5.0

**Comment by beginner-shuo** [comment]
Besides, the "tensor_size = SR.size(0)*SR.size(1)*SR.size(2)*SR.size(3) " should be changed to "tensor_size = SR.size(1)*SR.size(2)*SR.size(3)" since acc was calculated by "length += images.size(0)	acc = acc/length".

**Review by Lloyd-Pottiger — COMMENTED** [review]
Good job!

**Comment by Biel-Hammer** [comment]
> tested on pytorch==1.2.0 ISIC 2018 dataset first loop result[All default]: Epoch [1/150], Loss: 570.2478, [Training] Acc: 0.8824, SE: 0.7623, SP: 0.9695, PC: 0.7704, F1: 0.6987, JS: 0.5823, DC: 0.6987 [Validation] Acc: 0.8595, SE: 0.5696, SP: 0.9477, PC: 0.7414, F1: 0.5573, JS: 0.4587, DC: 0.5573 Best R2AttU_Net model score : 1.0160
> 
> tested on pytorch==1.5.0 ISIC 2018 dataset first loop result[All default]: Epoch [1/200], Loss: 454.9332, [Training] Acc: 0.9083, SE: 0.7878, SP: 0.9756, PC: 0.8060, F1: 0.7414, JS: 0.6310, DC: 0.7414 [Validation] Acc: 0.8288, SE: 0.3145, SP: 0.9982, PC: 0.6754, F1: 0.3814, JS: 0.2991, DC: 0.3814

but it only works on first epoch in my computer, with torch vision is 1.11.0
